src/common/request.js

```diff
@@ -54,6 +55,7 @@ export default class Request {
 
         request.end()
       } else {
+        const req = this._request
```

Comment:
It does support https as far as I know. https://www.w3.org/TR/XMLHttpRequest/#introduction